apps/bridge-dapp/src/utils/getCardTitle.ts

```diff
@@ -22,10 +22,13 @@ export const getCardTitle = (
     }
 
     default: {
-      status = 'In-Progress';
+      status = 'in Progress...';
```

Comment:
`In Progress...` or `in Progress...` 🤔 I think we should you `In Progress...` for consistency with other titles.

Reply:
I have updated all the card titles according to the design.
<img width="504" alt="Screenshot 2023-04-03 at 10 14 36 AM" src="https://user-images.githubusercontent.com/53374218/229584091-37e4d03d-8bb2-4980-a0ad-7ba393f38f21.png">